gulp-tasks/utils/build-browser-typescript.js

```diff
@@ -0,0 +1,73 @@
+const gulp = require('gulp');
+const path = require('path');
+const sourcemaps = require('gulp-sourcemaps');
+const rollup = require('rollup');
+const rollupStream = require('rollup-stream');
+const uglifyPlugin = require('rollup-plugin-uglify');
+const typescriptPlugin = require('rollup-plugin-typescript');
+const esMinify = require('uglify-es').minify;
+const rename = require('gulp-rename');
+const source = require('vinyl-source-stream');
+const buffer = require('vinyl-buffer');
+const util = require('util');
+const glob = util.promisify(require('glob'));
+
+const processScript = (scriptPath, relativePath, tsConfigPath, destDir) => {
+  const tsconfig = require(tsConfigPath);
+
+  return rollupStream({
+    rollup,
+    input: scriptPath,
+    output: {
+      format: 'iife',
+      sourcemap: true,
+      name: path.basename(scriptPath),
+    },
+    plugins: [
+      typescriptPlugin({
+        ...tsconfig.compilerOptions,
+        tsconfig: false,
```

Comment:
This ignores options in the tsconfig -- that doesn't seem right?

https://github.com/rollup/rollup-plugin-typescript#usage

Reply:
Oh sorry ignore me, missed the line right above.